data/TT/data.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
please fix the json validation errors. Trailing commas for the address keys

Reply:
thank you! fixed